securedrop/create-dev-data.py

```diff
@@ -56,8 +52,12 @@ def main(staging=False):
                                                 first_name="Clark",
                                                 last_name="Kent")
 
+        NUM_SOURCES = os.getenv('NUM_SOURCES', 2)
+        if NUM_SOURCES == "ALL":
+            # We want all strings
```

Comment:
Small request, please be a bit more explicit in the source code comment where the magic number 43 comes from :)